terracumber_config/tf_files/SUSEManager-4.3-build-validation-paygo-AWS.tf

```diff
@@ -0,0 +1,503 @@
+  sles_registration_code = var.SLES_REGISTRATION_CODE
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+  gpg_keys                = ["default/gpg_keys/galaxy.key"]
+  provider_settings = {
+    instance_type = "t3a.medium"
+  }
+  additional_packages = [ "chrony" ]
+}
+
+module "sles15sp4-sshminion" {
+  source             = "./modules/sshminion"
+  base_configuration = module.base.configuration
+  product_version    = "4.3-released"
+  name               = "minssh-sles15sp4"
+  image              = "sles15sp4o"
+  sles_registration_code = var.SLES_REGISTRATION_CODE
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+  provider_settings = {
+    instance_type = "t3a.medium"
+  }
+}
+
+
+module "sles15sp5-sshminion" {
+  source             = "./modules/sshminion"
+  base_configuration = module.base.configuration
+  product_version    = "4.3-released"
+  name               = "minssh-sles15sp5"
+  image              = "sles15sp5o"
+  sles_registration_code = var.SLES_REGISTRATION_CODE
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+  provider_settings = {
+    instance_type = "t3a.medium"
+  }
+
+}
+
+module "controller" {
+  source             = "./modules/controller"
+  name               = "ctl"
+  provider_settings = {
+    instance_type = "c6i.xlarge"
+  }
+  base_configuration = merge(module.base.configuration,
+    {
+      cc_username = var.SCC_USER
+      cc_password = var.SCC_PASSWORD
+    })
+
+
+  swap_file_size = null
+  no_mirror = true
+  is_using_build_image      = false
+  is_using_scc_repositories = false
+  is_using_paygo_server     = true
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+  server_instance_id  =  module.server.configuration.id
+
+
+  server_configuration    = module.server.configuration
+  proxy_configuration     = module.proxy.configuration
+
+  sle12sp5_paygo_minion_configuration       = module.sles12sp5-paygo-minion.configuration
+  sle15sp4_byos_minion_configuration       = module.sles15sp4-byos-minion.configuration
+  sle15sp5_paygo_minion_configuration       = module.sles15sp5-paygo-minion.configuration
+  sleforsap15sp5_paygo_minion_configuration = module.slesforsap15sp5-paygo-minion.configuration
+
+  sle12sp5_client_configuration    = module.sles12sp5-client.configuration
+  sle12sp5_minion_configuration    = module.sles12sp5-minion.configuration
+  sle12sp5_sshminion_configuration = module.sles12sp5-sshminion.configuration
+
+  sle15sp4_client_configuration    = module.sles15sp4-client.configuration
+  sle15sp4_minion_configuration    = module.sles15sp4-minion.configuration
+  sle15sp4_sshminion_configuration = module.sles15sp4-sshminion.configuration
+
+  sle15sp5_minion_configuration    = module.sles15sp5-minion.configuration
```

Comment:
Are we using the no-paygo minions anywhere for this PAYGO AWS Pipeline?
Would it make sense to test something not PAYG related on this pipeline?

Reply:
It doesn't cost much to add those minions and give us some feedback on paygo server. During my manual testing, I found some issues testing 'classic' minions with the paygo server. Just checking if the repositories are available is not enough in my opinion. Maybe we could limit it to minion. But it's really free to add those extra minions and I don't have server instability on AWS ( also with limited minions ).